### DISCUSSION TITLE
help in spliting Right to left spreads

### DISCUSSION THREAD

**Discussion by Fahad-Alsaidi** [pdfarranger/pdfarranger#1089] [Q&A] [answered]
I found that it is possible to split RTL spreads by pdfarrange in this bug [report](https://github.com/pdfarranger/pdfarranger/issues/236), but it is not obvious to me how.  I have this small file [Amr_Nami_2.pdf](https://github.com/pdfarranger/pdfarranger/files/15257720/Amr_Nami_2.pdf)
what are the right steps to split spreads in RTL order?
Thanks in advance.

**Comment by kbengs** [comment, marked as answer]
I am not sure I understand this right but maybe this way:
* select page 2 - 26
* split vertically
* select even pages and cut
* select the second page and paste as even pages
* select all and reverse order?

**Reply by Fahad-Alsaidi** [reply]
Thanks it works great.
